lib/sidekiq/scheduler.rb

```diff
@@ -234,7 +234,9 @@ def self.active_job_enqueue?(klass)
     #
     # @return [Hash]
     def self.prepare_arguments(config)
-      config['class'] = config['class'].constantize if config['class'].is_a?(String)
+      if config['class'].is_a?(String) && Object.const_defined?(config['class'])
+        config['class'] = Object.const_get(config['class'])
+      end
```

Comment:
I'm not really a fan of this as the behavior silently changes on whether the object exists in the load path (never mind if it is auto_loaded!)  My challenge is I don't know ActiveJob... I did some tracing and it definitely doesn't look like it is as 'simple' as not instantiating the class and just passing a hash to get queued by Sidekiq!  

Some more explicit options may be to:
* make an explicit configuration flag, such as `Sidekiq::Scheduler.enable_active_job = true` (default to true)
* print a `logger.warn` msg if the class cannot be constantized.

Any suggestions on how to thread this needle?